### PR TITLE
feat(EmbarcaderoMigrator): migrate posts slugs

### PR DESCRIPTION
**Pull request by kariae**

### CODE REVIEW

File: src/Command/General/EmbarcaderoMigrator.php [hunk 1]
```diff
@@ -1104,6 +1122,65 @@ function( $print_issue ) use ( $imported_original_ids ) {
...
 
+	/**
+	 * Callable for "newspack-content-migrator embarcadero-migrate-post-slugs".
+	 *
+	 * @param array $args array Command arguments.
+	 * @param array $assoc_args array Command associative arguments.
+	 */
+	public function cmd_embarcadero_migrate_post_slugs( $args, $assoc_args ) {
+		$story_csv_file_path   = $assoc_args['story-csv-file-path'];
+		$imported_original_ids = $this->get_comments_meta_values_by_key( self::EMBARCADERO_MIGRATED_POST_SLUG_META_KEY );
```
Comment (naxoc) [inline src/Command/General/EmbarcaderoMigrator.php:1133]: Should this be `get_posts_meta_values_by_key()` instead?

**Review by naxoc — COMMENTED** [review]
Thanks! Looks good. I just had one question.

File: src/Command/General/EmbarcaderoMigrator.php (hunk 1 above)
Comment (kariae) [reply src/Command/General/EmbarcaderoMigrator.php:1160]: Good call, fixing it, thanks!

**Review by naxoc — APPROVED** [review]
This looks good! 👍

**Pull request merged**